1.3/Dockerfile

```diff
@@ -1,13 +1,13 @@
 
-FROM erlang:19.0-rc2
+FROM erlang:18.3.3
```

Comment:
there are tags `erlang:18` and `erlang:18.3` always pointing the latest one, so use them will be better and less changes to this file in future